RecyclingStrategyCertificate/1.0.0/RecyclingStrategyCertificate.ttl

```diff
@@ -0,0 +1,79 @@
+#######################################################################
+# Copyright (c) 2023 T-Systems International GmbH
+# Copyright (c) 2023 SAP SE
+# Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft
+# Copyright (c) 2023 LRP Autorecycling Leipzig GmbH
+# Copyright (c) 2023 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e.V. für ihre Institute IPK und IPK
+# Copyright (c) 2023 tec4U-Solutions GmbH
+# Copyright (c) 2022 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+
+@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:2.0.0#>.
+@prefix bamm-c: <urn:bamm:io.openmanufacturing:characteristic:2.0.0#>.
+@prefix bamm-e: <urn:bamm:io.openmanufacturing:entity:2.0.0#>.
+@prefix unit: <urn:bamm:io.openmanufacturing:unit:2.0.0#>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix : <urn:bamm:io.catenax.shared.recycling_strategy_certificate:1.0.0#>.
+
+:Characteristic1 a bamm:Characteristic;
+    bamm:dataType xsd:string;
+    bamm:preferredName "UUIDv4Characteristic"@en;
+    bamm:description "A version 4 UUID is a universally unique identifier that is generated using random 32 hexadecimal characters."@en;
+    bamm:see <https://tools.ietf.org/html/rfc4122>.
+:catenaXId a bamm:Property;
+    bamm:preferredName "Catena-X Identifier"@en;
+    bamm:description "The Catena-X ID of the given part (e.g. the assembly), valid for the Catena-X dataspace."@en;
+    bamm:characteristic :CatenaXIdTraitCharacteristic;
+    bamm:exampleValue "urn:uuid:055c1128-0375-47c8-98de-7cf802c3241d".
+:CatenaXIdTraitCharacteristic a bamm-c:Trait;
+    bamm:description "Trait to ensure UUID v4 data format"@en;
+    bamm-c:baseCharacteristic :Characteristic1;
+    bamm-c:constraint :UUIDv4RegularExpression.
+:issuer a bamm:Property;
+    bamm:preferredName "issuer"@en;
+    bamm:description "BPMN of the business partner who creates this certificate"@en;
+    bamm:characteristic :IssuerCharacteristic;
+    bamm:exampleValue "BPNLBMW000000001".
+:IssuerCharacteristic a bamm:Characteristic;
+    bamm:dataType xsd:string;
+    bamm:preferredName "issuer characteristic"@en.
```

Comment:
I would write the preferred names upper case as they can be considered proper nouns

Reply:
Checked and altered first character for prefered name to uppercase throughout the whole file